charts/federation-v2/charts/controllermanager/templates/deployment.yaml

```diff
@@ -45,13 +33,13 @@ spec:
 {{- if .Values.resources }}
 {{ toYaml .Values.resources | indent 12 }}
 {{- end }}
-{{- if .Values.global.limitedScope }}
         env:
-        - name: POD_NAMESPACE
+        - name: FEDERATION_NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: FEDERATION_NAMESPACE
+{{- if .Values.global.limitedScope }}
```

Comment:
Is this block necessary, given that `POD_NAMESPACE` appears unused and `CLUSTER_REGISTRY_NAMESPACE` is intended to be set by `FederationConfig`?

Reply:
I will clean this because we donot use such 2 environment variables anymore.